assets/src/apps/Components.tsx

```diff
@@ -29,3 +30,4 @@ registerApplication('UserAccountMenu', UserAccountMenu, globalStore);
 registerApplication('DeliveryElementRenderer', DeliveryElementRenderer, globalStore);
 registerApplication('ECLRepl', ECLRepl, globalStore);
 registerApplication('SelectTimezone', SelectTimezone, globalStore);
+registerApplication('YoutubePlayer', YoutubePlayer, globalStore);
```

Comment:
Instead of just rendering an iframe to play the youtube, we're now using a react component that utilizes the google youtube iframe API, so that has to always be a client side implementation like the video player.